docs/blog/2019-06-21-profitable-agency-wordpress/index.md

```diff
@@ -0,0 +1,117 @@
+
+However, if your job is to build custom WordPress solutions for large sites, then never deal with them again, the cost savings I lay out may not matter as much to you.
+
+## Hosting for Gatsby sites starts at $0
+
+Last year I built a WordPress website for an event. I used a few popular membership and ecommerce plugins to get the job done.  The price of hosting and plugins were all obscured from the client. I was paid a fee, and then I turned around and paid for hosting and plugins, and pocketed the margin.
+
+I even collected an affiliate fee from the server host. Again, if site maintenance is your bread-and-butter, then you know this model. Every nickel counts.
+
+After the event, my client no longer needed the membership and ecommerce features. And they weren't enthusiastic about paying me for ongoing support. So, instead of haggling over a price, or handing over the project and losing their business forever, I turned the WordPress project into a static site that I host on Netlify.
+
+The hosting cost? [Zero dollars and zero cents.](https://www.netlify.com/pricing/)
+
+That project opened my eyes. The client had a low traffic website, and they barely used the membership and ecommerce functionality I added. Yet, I signed them up with my favorite WordPress host, with pricing that started at $115 a month, not including the cost of premium plugins. The next time I had a similar project, which was earlier this year, I grabbed Netlify from the beginning.
+
+The cheaper hosting for static sites, like the ones Gatsby can produce, can be a game changer for your business model. That's because the client is willing to pay for an all-inclusive package, and doesn't care if you use an inexpensive static site host, or a costly WordPress host. By making the right choice your profit can increase from day one, and accelerate if you keep the project over months or years.
+
+I'm old enough to remember using Movable Type and sneering at WordPress. I worked at Vibe Magazine and didn't choose WordPress as our CMS because at the time it couldn't handle our requirement for multiple authors. However, Movable Type eventually ceded their #1 position to WordPress, in large part due to hosting costs. Hosting for a PHP CMS like WordPress was cheaper and simpler than finding a home for the Perl-powered Movable Type.
+
+Static sites in general, and Gatsby in particular will win because the cost of getting started, and keeping going is so low. And as a bonus, static site hosting truly is more secure and simpler to use. It's rare that a new way of doing things ticks the boxes of FAST, GOOD, and CHEAP with no compromise. But that perfectly describes [the hosting options for Gatsby](https://www.gatsbyjs.org/docs/deploying-and-hosting/).
```

Comment:
```suggestion
Static sites in general, and Gatsby in particular will win because the cost of getting started, and keeping going is so low. And as a bonus, static site hosting truly is more secure and simpler to use. It's rare that a new way of doing things ticks the boxes of FAST, GOOD, and CHEAP with no compromise. But that perfectly describes [the hosting options for Gatsby](/docs/deploying-and-hosting/).
```
Making this link relative since it will already be on the Gatsby site.